### PR TITLE
HDDS-1571. Refactor ContainerPlacementPolicy.

### PR DESCRIPTION
**Pull request by timmylicheng**
#HDDS-1571
Refactor ContainerPlacementPolicy to Placement Policy for future exploit.

### CODE REVIEW

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/PlacementPolicy.java [hunk 1]
```diff
@@ -15,22 +15,22 @@
...
 
 import org.apache.hadoop.hdds.protocol.DatanodeDetails;
 
 import java.io.IOException;
 import java.util.List;
 
 /**
- * A ContainerPlacementPolicy support choosing datanodes to build replication
+ * A PlacementPolicy support choosing datanodes to build
  * pipeline with specified constraints.
```
Comment (ChenSammi) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/PlacementPolicy.java]: build pipeline or for container replica

**Comment by ChenSammi** [comment]
The path looks overall good. Just one small document issue.

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/PlacementPolicy.java (hunk 1 above)
Comment (timmylicheng) [reply hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/PlacementPolicy.java]: Updated

**Comment by ChenSammi** [comment]
+1. Committed to branch HDDS-1564.

**Pull request merged**